connector-template/test.rs

```diff
@@ -22,70 +21,402 @@ impl utils::Connector for {{project-name | downcase | pascal_case}} {
     fn get_auth_token(&self) -> types::ConnectorAuthType {
         types::ConnectorAuthType::from(
             connector_auth::ConnectorAuthentication::new()
-                .{{project-name | downcase }}
+                .{{project-name | downcase}}
                 .expect("Missing connector authentication configuration"),
         )
     }
 
     fn get_name(&self) -> String {
-        "{{project-name | downcase }}".to_string()
+        "{{project-name | downcase}}".to_string()
     }
 }
 
+static CONNECTOR: {{project-name | downcase | pascal_case}} = {{project-name | downcase | pascal_case}} {};
+
+// Cards Positive Tests
+// This test creates a payment using the manual capture flow (Non 3DS).
```

Comment:
```suggestion
// Creates a payment using the manual capture flow (Non 3DS).
```
as it is already a test, we can ignore `This test` here and also in other places